week-2/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,22 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0;
+}
+
+function isBetween5and10(num) {
+  return num >= 5 && num <= 10;
+}
+startsWithD;
+function isLessThan8Characters(name) {
+  return name.length < 8;
+}
+
+function startsWithD(letter) {
+  letter = "D";
+  return true;
```

Comment:
This function is just returning `true` for any input. It's also called with a string parameter not a letter so the name `letter` is confusing. This should be a boolean expression too.